src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/sge/SGEJobManager.groovy

```diff
@@ -0,0 +1,143 @@
+        super(executionService, parms)
+    }
+
+    @Override
+    SGECommand createCommand(GenericJobInfo jobInfo) {
+        throw new NotImplementedException()
+    }
+
+    SGECommand createCommand(BEJob job, List<ProcessingCommands> processingCommands, String command, Map<String, String> parameters, Map<String, Object> tags, List<String> dependencies, File logDirectory) {
+        SGECommand sgeCommand = new SGECommand(this, job, job.jobID.toString(), processingCommands, parameters, tags, null, dependencies, command, logDirectory)
+        return sgeCommand
+    }
+
+    @Override
+    SGECommand createCommand(BEJob job, String jobName, List<ProcessingCommands> processingCommands, File tool, Map<String, String> parameters, List<String> dependencies) {
+        throw new NotImplementedException()
+    }
+
+    SGECommand createCommand(BEJob job) {
+        return new SGECommand(this, job, job.jobName, [], job.parameters, [:], [], job.dependencyIDsAsString, job.tool?.getAbsolutePath() ?: job.getToolScript(), job.getLoggingDirectory())
+    }
+//    @Override
+//    public void addSpecificSettingsToConfiguration(Configuration configuration) {
+//        configuration.getConfigurationValues().add(new ConfigurationValue("RODDY_JOBID", "${JOB_ID-}"));
+//        configuration.getConfigurationValues().add(new ConfigurationValue("RODDY_SCRATCH", "/tmp/roddyScratch/${JOB_ID}"));
+//        configuration.getConfigurationValues().add(new ConfigurationValue("RODDY_AUTOCLEANUP_SCRATCH", "true"));
+//    }
+
+    @Override
+    ProcessingCommands parseProcessingCommands(String processingString) {
+        return convertPBSResourceOptionsString(processingString)
+    }
+
+    @Override
+    String getResourceOptionsPrefix() {
+        return "SGEResourceOptions_"
+    }
+
+    @Override
+    ProcessingCommands convertResourceSet(BEJob job, ResourceSet resourceSet) {
+        StringBuilder sb = new StringBuilder()
+        sb.append(" -V") //TODO Think if default SGE options should go somewhere else?
+        if (resourceSet.isMemSet()) {
+            String memoryInMB = resourceSet.getMem().toString(BufferUnit.M)
+            sb << " -l " << "s_data" << "=" << memoryInMB
+        }
+        if (resourceSet.isStorageSet()) {
+        }
+        return new PBSResourceProcessingCommand(sb.toString())
+    }
+
+    @Override
+    ProcessingCommands extractProcessingCommandsFromToolScript(File file) {
+        return null
+    }
+
+    @Override
+    String getStringForQueuedJob() {
+        return "qw"
+    }
+
+    @Override
+    String getStringForJobOnHold() {
+        return "hqw"
+    }
+
+    @Override
+    String getStringForRunningJob() {
+        return "r"
+    }
+
+    @Override
+    String getSpecificJobIDIdentifier() {
+        return "JOB_ID"
+    }
+
+    @Override
+    String getSpecificJobArrayIndexIdentifier() {
+        return PBS_ARRAYID
+    }
+
+    @Override
+    String getSpecificJobScratchIdentifier() {
```

Comment:
That's Roddy specific. It should not be in BE.

Reply:
No that is BE it is not Roddy

Reply:
It is implemented for the other managers as well. But we need to check if it is not already done in the super class.

Reply:
Well, it says "roddyScratch" and returns a path? What is there job manager specific. Rather seems actually cluster specific!

Reply:
Where do you see "roddyScratch"? Yes it is cluster specific.